guidelines/content-guidelines/screenshots.md

```diff
@@ -0,0 +1,46 @@
+# Screenshots
+
+As someone once said, a picture is worth a thousand words. Therefore, whenever you need to illustrate operations performed on the UI, feel free to use a screenshot to convey a lot of information visually.
+
+Follow these basic principles when placing a screenshot in your content:
```

Comment:
- when placing a screenshot -> `when you place screenshots`  (you use plural below so it is better to start here with it as well)